src/Core/PluginInterface.php

```diff
@@ -28,7 +28,7 @@ interface PluginInterface
      *
      * @var list<string>
      */
-    public const VALID_HOOKS = ['bootstrap', 'console', 'middleware', 'routes', 'services', 'events'];
+    public const VALID_HOOKS = ['bootstrap', 'console', 'middleware', 'routes', 'services', 'events', 'consoleEvents'];
```

Comment:
This would be the first hook which is camelCased. Is this fine or are there better alternatives?

Reply:
Why not have `events` be the one hook that is used for both web and console?

Reply:
Yeah, a separate hook for console events doesn't seem necessary.